fractal_server/app/api/v1/workflow.py

```diff
@@ -12,34 +12,53 @@
 """
 from fastapi import APIRouter
 from fastapi import Depends
+from fastapi import HTTPException
+from fastapi import status
+from sqlalchemy.exc import NoResultFound
+from sqlmodel import select
 
 from ...db import AsyncSession
 from ...db import get_db
 from ...security import current_active_user
 from ...security import User
 
+from ...models.workflow import Workflow
+
 router = APIRouter()
 
 
 @router.post("/")
 async def create_workflow(
+    workflow: Workflow,
```

Comment:
wrong input model: it should be `WorkflowCreate`. If it doesn't exist yet, please define it.